api/server/routes_create.go

```diff
@@ -99,5 +99,8 @@ func (s *Server) handleRouteCreate(c *gin.Context) {
 		return
 	}
 
+	// For any create just push the new route to the front of the list
```

Comment:
Is this comment really necessary? it seems quite self-evident for me that `Refresh` will push the new route to the front of the list.